package.json

```diff
@@ -34,7 +34,8 @@
     "sass-loader": "^10.1.1",
     "svg-inline-loader": "^0.8.2",
     "v-mask": "^2.2.4",
-    "vuepress": "^1.8.2"
+    "vuepress": "^1.8.2",
+    "vuepress-plugin-flowchart": "^1.4.3"
```

Comment:
does not look like this plugin is being actively developed, decent chance we'd have to do the work to update it to vue3 when that time comes. not a huge issue just a maintenance cost to be aware of.

Reply:
Yea that is something to consider - I'll keep it in for now as it lets us get the flowcharts articulated quickly - but these could probably be converted to images without too much work